### PR TITLE
Feature/refactor

### PR DESCRIPTION
**Pull request by dancoto**
Reorganized app structure
Switch to SASS for styling
Changed default test runner to Jest
Added lazy loading module routing
Added proxy for API calls

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-    <a mat-tab-link routerLink="/requests">Requests</a>
-    <a class="nav-fill-space"></a>
-  </nav>
-<!--  <app-feedtable></app-feedtable>-->
-  <router-outlet></router-outlet>
+    <nav mat-tab-nav-bar>
+        <a class="nav-fill-space"></a>
+        <a mat-tab-link routerLink="/feeds" routerLinkActive="active">Feeds</a>
+        <a mat-tab-link routerLink="/downloads" routerLinkActive="active"
+            >Downloads</a
```
Comment (tnessy) [inline src/app/app.component.html]: Some weird new line formatting going on here... intentional?

Comment (dancoto) [reply src/app/app.component.html]: Default prettier rules. This is intended and will be standardized across all UI for consistency.

Comment (dancoto) [reply src/app/app.component.html]: https://prettier.io/

**Review by tnessy — APPROVED** [review]

File: src/app/app.component.html (hunk 1 above)
Comment (tnessy) [reply src/app/app.component.html]: So `>Downloads</a` is an acceptable styling? TBH, it's really hard to read.

Comment (dancoto) [reply src/app/app.component.html]: Updated prettier rules. It should be easier on the eyes.

**Pull request merged**